StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Views/BacsDDMandateView.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  BacsDDMandateView.swift
+//  StripePaymentSheet
+//
+//  Created by David Estes on 9/6/23.
+//
+
+@_spi(STP) import StripeUICore
+import SwiftUI
+import UIKit
+
+// Reusing these colors from Link:
+private let headerColor = Color(UIColor.linkPrimaryText)
+private let secondaryTextColor = Color(UIColor.linkSecondaryText)
+private let borderColor = Color(UIColor.linkSeparator)
+private let backgroundColor = Color(UIColor.linkSecondaryButtonBackground)
+
+private let shadow1Color = Color(red: 18/255, green: 42/255, blue: 66/255, opacity: 0.04)
+private let shadow2Color = Color(red: 0/255, green: 0/255, blue: 0/255, opacity: 0.04)
+
+struct BacsDDMandateView: View {
```

Comment:
This view should get it's colors from the appearance API imo

Reply:
We could but I'm a little worried about it, maybe unnecessarily. PE on the web doesn't, and they had some pretty strict warnings around the design, which I interpreted as including the specific contrast of the colors and such.